compair/static/modules/assignment/assignment-form-partial.html

```diff
@@ -45,6 +48,7 @@ <h1 ng-if="method == 'copy'">Duplicate: {{originalAssignment.name}}</h1>
 
         <div>
             <label for="importFile">Related Attachment</label>
+            <p ng-show="!assignment.file && assignment.existingFile" class="alert alert-warning"><strong>Save this assignment to save changes made to your attachment</strong>, or leave this screen to keep the attachment that appeared when you started editing.</p>
```

Comment:
Ditto. Should the alert also be displayed when:
- adding an attachment to a draft answer that has no attachment?
- rotating an attached image?

Reply:
Good questions, address these below.